lib/main_common.pm

```diff
@@ -1709,8 +1709,9 @@ sub load_extra_tests_docker {
     loadtest "containers/docker";
     loadtest "containers/docker_runc";
     loadtest "containers/container_base_images";
-    loadtest "containers/docker_image" if (!is_public_cloud && (is_sle(">=12-sp3") || is_opensuse));
-    loadtest "containers/docker_compose" unless is_sle('<15');
+    loadtest "containers/docker_image"   if (!is_public_cloud && (is_sle(">=12-sp3") || is_opensuse));
```

Comment:
Condition needs deduplicating

Reply:
Can you suggest me how? I don't see any common condition here? Every test module has a bit different requirements.

Reply:
```
if(!is_public_cloud && (is_sle(">=12-sp3") || is_opensuse)) {
    loadtest "containers/docker_image"
    loadtest "containers/container_diff"
}
```